.gitignore

```diff
@@ -5,5 +5,6 @@ yarn-error.log
 /cypress/screenshots/
 /__sapper__/
 /static/main.css
-
-.now
\ No newline at end of file
+.vscode/
```

Comment:
I think the vscode settings should be check in.
There are task to run, how to build and all this things

Reply:
I've got the location of my node in there:

```
{
  "svelte.language-server.runtime": "/Users/vanessa.boehner/.nvm/versions/node/v12.16.1/bin/node",
  "svelte-intellisense.trace.server": "messages"
}
```

Reply:
Thats really a bad idea from the svelte language server...
You should probably move that property into your userfile configuration. vscode has different levels of configuration and everything project specific should be in the project and be checked in.

Reply:
yeah makes sense, I've copied it a bit blindly. thanks

Reply:
.vscode is removed from gitignore